examples/js/shaders/VolumeShader.js

```diff
@@ -0,0 +1,320 @@
+// Author: Almar Klein
+// Description: Shaders to render 3D volumes using raycasting.
+// The applied techniques are based on similar implementations in the Visvis and Vispy projects.
+// This is not the only approach, therefore it's marked 1.
```

Comment:
nit: We generally use this style

    /**
     * @author Almar Klein / url
     *
     * ....
     */